opencsp/app/sofast/test/test_DisplayShape.py

```diff
@@ -87,17 +88,51 @@ def test_distorted2D(self):
 
     def test_distorted3D(self):
         # Instantiate display object
-        v_cam_screen_screen = Vxyz((0, 0, 1))
-        r_screen_cam = Rotation.from_rotvec(np.array([0.0, 0.0, 0.0]))
         name = 'Test DisplayShape'
-        disp = DisplayShape(v_cam_screen_screen, r_screen_cam, self.grid_data_3D, name)
+        disp = DisplayShape(self.grid_data_3D, name)
 
         # Perform calculation
         calc = disp.interp_func(self.test_Vxy_pts)
 
         # Test
         np.testing.assert_allclose(calc.data, self.exp_Vxyz_disp_pts.data, rtol=0, atol=1e-7)
 
+    def test_save_load_hdf_dist_3d(self):
+        # Instantiate display object
+        name = 'Test DisplayShape'
+        disp = DisplayShape(self.grid_data_3D, name)
+        file = join(self.save_dir, 'test_display_shape_dist_3d.h5')
+
+        # Save
+        disp.save_to_hdf(file)
+
+        # Load
+        DisplayShape.load_from_hdf(file)

```

Comment:
I suggest expanding this test method to compare the saved and loaded versions of the DisplayShape object.

Reply:
Good idea, updated!